src/components/inputs/rich-text-input/editor.js

```diff
@@ -103,15 +103,9 @@ const Editor = props => {
 
 // eslint-disable-next-line react/display-name
 const renderEditor = (props, editor, next) => {
-  const children = next();
-
-  const transformedChildren = {
-    ...children,
-    props: {
-      ...children.props,
-      tagName: 'output',
-    },
-  };
+  const children = React.cloneElement(next(), {
+    tagName: 'output',
+  });
```

Comment:
This is far nicer than the fn on the right regardless of having to switch over to an output tag 🌟🤙🏼